docs/admin/install.rst

```diff
@@ -650,6 +650,27 @@ After enabling it, you should set :setting:`ENABLE_HTTPS` in the settings:
    :setting:`ENABLE_HTTPS`,
    :ref:`production-site`
 
+
+Set properly SECURE_HSTS_SECONDS
+++++++++++++++++++++++++++++++++
+
+If your site is served over SSL, you have to consider setting a value for :setting:`SECURE_HSTS_SECONDS`
+in the settings.py and enabling HTTP Strict Transport Security.
```

Comment:
Setting SECURE_HSTS_SECONDS is way to enable HSTS, so it should probably be:

```suggestion
in the settings.py to enable HTTP Strict Transport Security.
```

Reply:
Fixed.

Reply:
@nijel  - I have updated the PR. Please review.

Reply:
![image](https://user-images.githubusercontent.com/8374949/82111390-d2730d80-96f9-11ea-8923-9f5f72e9cf49.png)